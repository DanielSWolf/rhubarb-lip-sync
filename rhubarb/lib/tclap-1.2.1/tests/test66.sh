#!/bin/sh

# this tests whether all required args are listed as
# missing when no arguments are specified
# failure  
../examples/test12  > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test66.out; then
	exit 0
else
	exit 1
fi

