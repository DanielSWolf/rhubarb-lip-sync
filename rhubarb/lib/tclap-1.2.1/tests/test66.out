PARSE ERROR:  
             Required argument missing: vect

Brief USAGE: 
   ../examples/test12  -v <3D vector> ...  [--] [--version] [-h]

For complete USAGE and HELP type: 
   ../examples/test12 --help

